bot/bot.go

```diff
@@ -59,7 +59,7 @@ func (b *Bot) Answer(question *query.Question) ([]query.Answer, error) {
 	if runExt != "" && b.Extension != nil {
 		reply, err = b.Extension.RunExtFunc(question, runExt, b.DB, machine)
 		if err != nil {
-			return nil, err
+			log.Error(err) // return nil, err
```

Comment:
Since errors from `RunExtFunc` will be converted into error replies, I think the error returned from this `Answer` function can be deprecated. What do you think @ryancurrah ?

Reply:
Done.